init/Bootstrap.js

```diff
@@ -134,6 +133,19 @@ class Bootstrap extends EventEmitter {
          performance.setContext("user", {
             id: userInfo.id,
          });
+      } else {
+         let { options: tenantConfig } = Config.tenantConfig();
```

Comment:
I haven't seen this javascript before.  I'm familiar with `let { tenantConfig } = Something`, but is this format saying that `Config.tenantConfig()` returns a `.option` value that you want stored in a new `tenantConfig` variable?

Reply:
Yes, that's right